releasewarrior/templates/fennec/release.json.tmpl

```diff
@@ -19,7 +19,7 @@
             {
                 "alias": "pushapk",
                 "description": "run pushapk (out of tree relpro)",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/old-how-tos/relpro.md#3-push-to-releases-dir-mirrors",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Push-to-Google-Play#what-to-do",
```

Comment:
until we get to 59.0 we will want https://github.com/mozilla-releng/releasewarrior-2.0/blob/10d9b968cfb1fd812dd3425c77e2638ba29ead4c/old-how-tos/fennec-temp-relpro.md#publish-release right?

Reply:
I think so.